content/docs/concepts/staking/_index.md

```diff
@@ -198,19 +200,32 @@ For example, an unbonding scenario with an epoch period of 30 blocks, an unbondi
+Staking transitions are changes to stake bonded to validators caused by:
+
+- stake bonding and unbonding operations submitted by stake delegators
+- [slashing](/concepts/staking/#slashing) penalties applied by protocol for proven validator faults whilst a member of the consensus committee. 
+
+Bonding and unbonding requests submitted during an epoch are processed and committed to state in the next available block, but the effect of such staking transitions is only applied at epoch end. Until epoch end staking transitions are maintained in memory in `BondingRequest` and `UnbondingRequest` data structures. They can be read by listening for and viewing `NewBondingRequest` and `NewUnbondingRequest` events emitted by the [`bond()`](/reference/api/aut/#bond) and [`unbond()`](/reference/api/aut/#unbond) functions of the Autonity Protocol Contract.
 
-Staking transitions are created by stake bonding and unbonding operations. Stake token is liable to protocol application of slashing penalties for accountability and omissions faults applied to validators whilst locked in the `bonded` and `unbonding` states.
+In Autonity's AFD protocol, slashable faults are likewise processed throughout an epoch and any staking transitions caused by stake slashing are applied to unbonding and bonded stake at epoch end according to Autonity's [Penalty-Absorbing Stake (PAS)](/concepts/staking/#penalty-absorbing-stake-pas) model. i.e. self-bonded stake is slashed in priority to delegated stake. 
 
-Bonding and unbonding requests submitted during an epoch are processed and committed to state in the next available block, but the effect of such staking transitions is only applied at epoch end. Until epoch end they are maintained in memory in the `Staking` data structure and can be viewed using the `getBondingReq()`, `getUnbondingReq()` functions of the Autonity Protocol Contract. (See the [Autonity Contract Interface](/reference/api/aut/).)
+As noted in [Protocol assets](/concepts/protocol-assets/), Newton and Liquid Newton token can be in different states. Bonded and unbonding stake is liable to protocol application of [slashing](/concepts/staking/#slashing) penalties:
 
+- Newton whilst locked in [states](/concepts/protocol-assets/newton/) `bonded` and `unbonding`.
+- Liquid Newton in [states](/concepts/protocol-assets/liquid-newton/) `locked` and `unlocked`.
+
+Whilst stake is unbonding the protocol tracks the relative ownership of stake in the delegated and self-bonded unbonding pools via a "share" mechanism so that the PAS slashing priority may be correctly applied to the unbonding stake and the correct amount of delegated or self-bonded stake be released at the end of the unbonding period.
+
+{{< alert title="Info" >}}
+Metadata stating the unbonding pools and tracking "shares" amounts are returned as part of the response when querying for a validator. See the [`getValidator()`](/reference/api/aut/#getvalidator) response object which contains fields for:
```

Comment:
The info in the validator object provides the total amount of shares and unbonding stake, however the single shares for each unbonding request are stored in the `UnbondingRequest` object

Reply:
Yes, that's covered in the paragraph preceding this note at Line 207:

But it could be clearer here. I will edit the Info note and reword to add this.

Reply:
See commit d80e806